featurebyte/middleware.py

```diff
@@ -168,6 +169,13 @@ async def __aexit__(self, *exc: Dict[str, Any]) -> bool:
     else HTTPStatus.NOT_FOUND,
 )
 
+ExecutionContext.register(
+    ColumnNotFoundError,
+    handle_status_code=lambda req, exc: HTTPStatus.UNPROCESSABLE_ENTITY
+    if req.method == "POST"
```

Comment:
POST or PATCH?

Reply:
We want 404 for PATCH so this should not be POST, we can keep it in case we need to handle this error during document creation